index.html

```diff
@@ -95,6 +105,15 @@ <h3 class="skill-title">Preprocessors</h3>
         </ul>
       </section>
```

Comment:
- [ ] [REQUIRED] As stated in the project requirements, you must replicate the design as much as possible (e.g., font, colors, images, tests, margins) using the info from the Figma templates. In this section, there are 3 columns (Languages, Frameworks, and Skills respectively) but you have added another column, now they are four. Please fix that so as to match the original design.